grpclog/component.go

```diff
@@ -87,68 +88,60 @@ func init() {
 				fmt.Fprintf(os.Stderr, "error: could not parse %v value '%v', unrecognized key-value pair '%v'\n", envName, envVar, varPair)
 				os.Exit(1)
```

Comment:
Still see os.Exit here.

Reply:
The os.Exit(1)s in the init have been removed.